checker/tests/nullness/Issue2432.java

```diff
@@ -0,0 +1,112 @@
+// Test case for issue 2432:
```

Comment:
Above this in the PR I see the addition of two files: `checker/annotation-tools` and `checker-stubparser`.
However, I can't edit nor delete those files.
Can you remove those changes?

Reply:
I cannot remove either, every time I tried to delete or edit github would return a 404 page.